components/core/src/clp_s/ReaderUtils.cpp

```diff
@@ -103,15 +103,15 @@ std::shared_ptr<ReaderUtils::SchemaMap> ReaderUtils::read_schemas(std::string co
             throw OperationFailed(error_code, __FILENAME__, __LINE__);
         }
 
-        std::set<int32_t>& schema = schemas[schema_id];
+        std::vector<int32_t>& schema = schemas[schema_id];
```

Comment:
```suggestion
        auto& schema = schemas[schema_id];
```
So that we don't need to care about the underlying type.